test/e2e/page-objects/pages/permission/permission-list-page.ts

```diff
@@ -0,0 +1,50 @@
+import { Driver } from '../../../webdriver/driver';
+
+/**
+ * Represents the permissions list page.
+ * This page allows users to view permissions for connected sites.
+ */
+class PermissionListPage {
+  private driver: Driver;
+
+  private readonly permissionsPage = '[data-testid="permissions-page"]';
+
+  constructor(driver: Driver) {
+    this.driver = driver;
+  }
+
+  async check_pageIsLoaded(): Promise<void> {
+    try {
+      await this.driver.waitForSelector(this.permissionsPage);
+    } catch (e) {
+      console.log(
+        'Timeout while waiting for permission list page to be loaded',
+        e,
+      );
+      throw e;
+    }
+    console.log('Permission list page is loaded');
+  }
+
+  /**
+   * Open permission page for site
+   *
+   * @param site - Site to open
+   */
+  async openPermissionPageForSite(site: string): Promise<void> {
+    console.log('Open permission page for site', site);
+    await this.driver.clickElement({ text: site, tag: 'p' });
```

Comment:
maybe we can add this into the selectors above? 

`{ text: site, tag: 'p' }`

Reply:
ohhh you are right! I thought it was a regular selector, now I see it' has a variable passed :pray: I think it make sense as it is!